xml/deployment_additional.xml

```diff
@@ -17,9 +17,9 @@
  </info>
  <para>
   After you deploy your cluster you may want to install and use other software
-  that by default are not shipped with &productname;. This chapter gives you
+  that is not shipped with &productname; by default. This chapter gives you
   detailed instructions how to deploy e.g. the &kube; dashboard or how to set
-  up a DNS server.
+  up Kubernetes DNS for service discovery.
```

Comment:
Please use the entity "&kube" instead of "Kubernetes" here. It will be resolved to "Kubernetes" through entity-decl.ent.